### ISSUE TITLE
VendorPrefixes and text-size-adjust not working properly?

### ISSUE THREAD

**Issue by epavletic** [causes/scss-lint#362]
I'm trying to declare `-webkit-text-size-adjust: 100%` in my stylesheet, but scss-lint keeps tripping on the declaration. If I keep the vendor prefix, it'll throw a `VendorPrefixes: Avoid vendor prefixes.`-error, but if I remove it (I use autoprefixer, so this is how I'd like to do it) then scss-lint gives me the `PropertySpelling: Unknown property text-size-adjust` error instead.

I've tried every possible combination of configuration (listed [here](https://github.com/causes/scss-lint/blob/master/lib/scss_lint/linter/README.md#vendorprefixes) ) – adding a `base` indentifier_list and excluding text-size-adjust, but to no avail.

Am I doing it wrong?

scss-lint 0.33.0 via grunt-scss-lint 0.3.4, but the errors show when I run scss-lint as a gem as well.

**Comment by sds** [comment]
Hey @epavletic,

We didn't have `text-size-adjust` in our list of default properties. I've fixed this in 4f08034.

You can either download the latest `master` branch of the gem, or you can add `text-size-adjust` to the `extra_properties` option of the `PropertySpelling` linter to fix the issue until a new version is released.

```yaml
linters:
  PropertySpelling:
    extra_properties:
      - text-size-adjust
```

**Comment by epavletic** [comment]
Thanks Shane, I could've swore I saw `text-size-adjust` in the [base.txt-list](https://github.com/causes/scss-lint/blob/master/data/prefixed-identifiers/base.txt) but maybe I've misunderstood how it's implemented. Anyways, thank you for the `extra_properties`-syntax; will give it a spin!

Much ❤ for SCSS-Lint.
